entry_types/scrolled/package/src/frontend/Entry.module.css

```diff
@@ -5,3 +5,9 @@
   background-color: #000;
   color: #fff;
 }
+
+.Entry > div:first-child {
+  /* Let content begin below navigation bar. Navigation bar has zero
+     height to let first chapter start at the very top. */
```

Comment:
**praise:** This is a well-chosen use for an inline comment. The line that it describes might change if we just need to adjust the value slightly, which could make the original commit message harder to find